core/rootcling_stage1/CMakeLists.txt

```diff
@@ -21,3 +21,4 @@ ROOT_EXECUTABLE(rootcling_stage1 src/rootcling_stage1.cxx
                               NOINSTALL)
 set_target_properties(rootcling_stage1 PROPERTIES RUNTIME_OUTPUT_DIRECTORY src)
 add_dependencies(rootcling_stage1 CLING LLVMRES)
+target_link_libraries(rootcling_stage1 gcc_eh)
```

Comment:
This is hacky, I don't want to merge it to master. However, it will allow you to reach further in the compilation while a proper fix is not found.

Reply:
Could you update the commit log by mentioning the error message that this commit is fixing?

Reply:
I'm going to drop this commit. It's just a hack I put in place to get further in the compilation. I didn't really mean to put it in this PR. I had to add that to be able to link rootcling_stage1 since libgcc was missing (the error was undefined symbol `__register_frame` or something like that).